script/Ciel/Day 14/14_01B_01_CIEL14B_KRAKE.txt

```diff
@@ -352,117 +378,128 @@ Its body is completely annihilated by Ciel's black keys.
 [sha:c218fb10fbbc0f09dbf3cf4177750f02192ec626]{
 -- Page 32, Offset 38416.
 -- 表面の９６パーセントが貫かれ、焼かれ、はじけ飛び、再生困難な状況になり、
-Ninety-six percent of its surface is penetrated, burned, and burst open, putting it in a difficult condition to regenerate.
+Ninety-six percent of its surface is pierced, burned and torn open, rendering regeneration impossible.
 }
 [sha:2accfa252954b98f5278b835652f813bf1693c28]{
 -- Page 32, Offset 38417.
 -- 接近戦用の左の巨腕は<関節技|アームロック>によってもぎ取られ、
-Its massive left arm specialized for close combat is torn off through an armlock.
+It's massive left arm,
+ specialized for close combat,
+ is put in an armlock and ripped off.
 }
 [sha:22bdd87291a3d5f70bbd3a7a847cd66dc40e95ad]{
 -- Page 32, Offset 38418.
 -- とどめとばかりに、無防備になった大胸骨に50トンもの<正拳|インパクト>がたたき込まれた。
-And to top it all off, a fifty-ton impact is delivered to its massive sternum that was unprotected.
+Then, as if to finish it off,
+ she delivers a fifty-ton impact to its gargantuan, now-unprotected, sternum.
 }
 [sha:7c50aab13014cb6a8b4cc18ffa1abdc88b506df6]{
 -- Page 33, Offset 38419.
 -- その外見通り、不死身と思われた怪物もここまで破壊されては退場する他ない。
-As one could see through its appearance, the monster that was considered immortal had no choice but to leave after such devastation.
+Just looking at the extent of the destruction,
+ it's clear that even monsters thought to be immortal would be forced to leave the stage after suffering such devastation.
 }
 [sha:6798ceb12b6d16086592e89b01f6c0f40932e952]{
 -- Page 33, Offset 38420.
 -- 真の不死―――限度のない復元者であるシエルと<耐久力|タフネス>で競った時点で、怪物は敗北していたと言える。
-True immortality―――One could argue that the monster had been defeated the moment it competed in toughness against Ciel, who was a limitless regenerative being.
+One could say that this monster had already lost the moment it tried to compete on toughness against Ciel――
+someone possessed of a truly undying body, and whose restoration knows no bounds.
 }
 [sha:cd3bb53cc0fdc446f7b501658704b08c007a3369]{
 -- Page 34, Offset 38421.
 -- 「ハァ……ハァ……ハァーーーー…………」
-"Hah... hah... hah......"
+"%{i}Hah%{/i}... %{i}hah%{/i}... %{i}haaah%{/i}..."
 }
 [sha:7e2881278b502f05da62140259dfcd818c6ff0bc]{
 -- Page 34, Offset 38422.
 -- とはいえ、この戦闘は彼女にとっても予想外のダメージを与えていた。
-Nevertheless, the fight inflicted far more damage to her than she had anticipated.
+Even so, the fight inflicted far more damage to her than she had anticipated.
 }
 [sha:c1f3107894233f5535c97f31144cc2060f3b9552]{
 -- Page 34, Offset 38423.
 -- 体力、肉体の損傷ではない。
-The damage was not on her body or her strength,
+Not against her body, or her stamina―――
 }
 [sha:3029e60c2760369931fa9b9fc53f79f0dbeae1c8]{
 -- Page 34, Offset 38424.
 -- 精神的な傷を、である。
-but rather on her mind.
+But rather against her mind.
```

Comment:
inflicting damage to "stamina" is kinda weird.
Probably just drop it? I think the point is made regardless.

Some other simplifications 
>Not to her body―――

>But to her mind.